client/src/components/app.jsx

```diff
@@ -6,21 +6,24 @@ import ImageCarousel from "./imageCarousel.jsx";
 import Modal from "./modal.jsx";
 // import styles from "../style.module.css";
 
+let randomNum = Math.floor(Math.random() * 10000000);
+
 class CaroApp extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            product : {},
-            displayedImageObj : {}
+            product : 'Cheapest Black T Shirt',
+            images: [],
+            displayedImage: ''
         }
     }
 
     componentDidMount() {
-        axios.get('/api/products/')
+        axios.get(`/api/products/${randomNum}`)
           .then((response) => {
               this.setState({
-                  product: response.data[0],
-                  displayedImageObj: response.data[0].photos[0]
+                  images: response.data,
+                  displayedImage: response.data[0].photo_url
```

Comment:
ohhh i like this idea. This allows you to display a random product each time... Gonna def use this myself. Good thinking!